internal/backends/testing/Cargo.toml

```diff
@@ -19,3 +19,4 @@ path = "lib.rs"
 i-slint-core = { version = "=0.2.5", path = "../../../internal/core" }
 
 image = { version = "0.24.0", default-features = false, features = ["png", "jpeg"] }
+euclid = "0.22.1"
```

Comment:
do we need it, or can we not use the re-export?

Reply:
Good point, we don't need it. I've changed it also with all the other crates.